src/app/page.tsx

```diff
@@ -12,63 +12,69 @@ import styles from './_styles/Homepage.module.scss';
 import { InvitationCta } from './components/InvitationCta/InvitationCta';
 const DibbsSection = () => (
   <ContentContainer align>
-    <ImageCard
-      imageUrl={`${basePath}/images/homepage-1.jpeg`}
-      imageAlt=""
-      imageFirst={true}
-      imageStyle={{
-        transform: 'scale(1.4) translate(-8%, 3%)',
-      }}
-    >
-      <Heading className="min-w-full text-center xl:text-left">
-        Introducing Data Integration Building Blocks
-      </Heading>
-      <div className="flex flex-col gap-2">
-        <Paragraph className="m-0 max-w-[39.7rem] p-0">
-          {homeContent.dibbs.description}
-        </Paragraph>
-        <ul className="text-base font-semibold leading-relaxed text-[#224a58]">
-          {homeContent.dibbs.benefits.map((benefit, index) => (
-            <li className="min-w-full" key={`benefit-${index}`}>
-              {benefit}
-            </li>
-          ))}
-        </ul>
-      </div>
-    </ImageCard>
-  </ContentContainer>
-);
-
-const ValueSection = () => {
-  const { valueSection } = homeContent;
-
-  return (
-    <ContentContainer sectionClasses={styles.valueSection} align>
+    <section>
```

Comment:
The diff makes this look like a lot of changes but I'm just wrapping these section components in `section` tags.

Reply:
Same thing for the other files below.